packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-widget.jsx

```diff
@@ -53,7 +53,9 @@ const ContractTypeWidget = ({ is_equal, name, value, list, onChange, languageCha
         if (selected_item && selected_item.value !== value) {
             onChange({ target: { name, value: selected_item.value } });
         }
-    }, [selected_item, onChange, name, value]);
+        // don't include value into the dependency array, it'll breake TradeTypeTab component
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
@maryia-deriv Please avoid ignoring `eslint` rules 🙏🏻 

In most cases when we have to ignore `useEffect` dependencies that means we are doing something wrong, For example here in this case, We don't need a `useEffect` at all, We can just call the `onChange` function inside of the `handleSelect` method.

Reply:
@farzin-deriv, I tried to implement this suggestion, but face a problem: without that `useEffect` we can't switch between different contract types :( Not between `Short/Long`  or `Call/Put`, but between `Turbos`, `Vanillas` and etc.
 While working on `Turbos` we try to refactor some old code if needed, but we can't improve it all as it might take too much time and qa attention. Yes, adding `eslint-disable` isn't a good practise, I agree 100%, but in other case we will have to refactor the whole component. 
 🙇🏼‍♀️ 🙇🏼‍♀️ 🙇🏼‍♀️ 🙇🏼‍♀️ 🙇🏼‍♀️